lighthouse-core/test/results/sample_v2.json

```diff
@@ -3172,6 +3172,7 @@
     "disableStorageReset": false,
     "disableDeviceEmulation": false,
     "emulatedFormFactor": "mobile",
+    "channel": "cli",
```

Comment:
Do we want this info in LR by adding it to the proto definition?

Reply:
Related: If a proto property exists, but it's always the default, does anyone notice it?

😆

Reply:
@exterkamp Would I just add it as as field number 4 here? https://github.com/GoogleChrome/lighthouse/blob/master/proto/lighthouse-result.proto#L141